ADA/databricks_workflows.qmd

```diff
@@ -8,19 +8,19 @@ title: "Databricks Workflows"
 
 ------------------------------------------------------------------------
 
-Workflows allow you to build complex data pipelines by chaining together multiple scripts, queries, notebooks and logic. They can be used to build Reproducible analytical pipelines (RAP) that can be re-run with different parameters and have all inputs and outputs audited automatically. Other recommended uses of workflows are any data modelling tasks such as cleaning your source data and collating it into a more analytically friendly format in your modelling area.
+Workflows allow you to build complex data pipelines by chaining together multiple scripts, queries, notebooks and logic. They can be used to build Reproducible Analytical Pipelines (RAP) that can be re-run with different parameters and have all inputs and outputs audited automatically. Other recommended uses of workflows are any data modelling tasks such as cleaning your source data and collating it into a more analytically friendly format in your modelling area.
 
 Each step in a workflow is referred to as a task and each task has dependencies. They are accessible through the 'Workflows' link on the left hand menu of the DataBricks UI.
 
 ![](/images/ada-workflow-menu.png)
 
-Parameters can be set either at a workflow level or a task level and referred to in your scripts / notebooks, allowing you to reuse tasks/workflows for similar operations.
+Parameters can be set either at a workflow level or a task level and referred to in your scripts / notebooks, allowing you to reuse tasks / workflows for similar operations.
 
 Each task can have dependencies on other tasks and can be set to only run under certain conditions, for example all of the previous tasks have completed successfully. These can be configured when tasks are added to the workflow through the user interface.
 
-Workflows and tasks can also be configured to send notifications to users upon success/failure. These can be configured from the Workflow and Task user interfaces.
+Workflows and tasks can also be configured to send notifications to users upon success / failure. These can be configured from the Workflow and Task user interfaces.
 
-Workflows also come with robust support for git/DevOps repositories and can be set to run from a specific repo, branch, commit or tag.
+Workflows also come with robust support for Git / Azure DevOps repositories and can be set to run from a specific repo, branch, commit or tag.
```

Comment:
Don't need to specify Azure DevOps if we're just saying Git?

Or if we want to, we should say `GitHub or Azure DevOps`?